zuul.d/jobs.yaml

```diff
@@ -108,3 +108,12 @@
       nodes:
         - name: builder
           label: ubuntu-xenial-builder
+
+- job:
+    name: contrail-vnc-docker-provision-centos74
+    voting: False
+    run: playbooks/docker/centos74-provision
+    nodeset:
+      nodes:
+        - name: builder
```

Comment:
This is no longer a "builder" but rather a "contrail-aio" (all-in-one) or something similar.